test/mocks/getPackageResponse.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
Looks good. BTW I'm increasingly thinking we should have a bunch of core mock data somewhere we can reuse over and over again e.g. in github.com/datopian/ckan-mock-data and we submodule it into projects somewhere ... (but this is an aside).